app/tests/src/test_citations.py

```diff
@@ -155,7 +156,6 @@ def test_tree_based_chunk_splitter():
 
 def test_replace_citation_ids():
     assert replace_citation_ids("No citations", {}) == "No citations"
-    assert replace_citation_ids("Hallucinated.(citation-1)", {}) == "Hallucinated."
```

Comment:
No longer needed now that `remove_unknown_citation_ids()` is called before calling this.